site/package.json

```diff
@@ -3,11 +3,14 @@
   "version": "1.0.0",
   "description": "Docs and examples for Svelte",
   "scripts": {
-    "dev": "npm run copy-workers && sapper dev",
+    "dev": "run-p update:* copy-workers && sapper dev",
     "copy-workers": "rm -rf static/workers && cp -r node_modules/@sveltejs/svelte-repl/workers static",
     "migrate": "node-pg-migrate -r dotenv/config",
-    "sapper": "npm run copy-workers && sapper build --legacy",
-    "update": "node scripts/update_template.js && node scripts/get-contributors.js && node scripts/update_whos_using.js",
+    "sapper": "npm run update && npm run copy-workers && sapper build --legacy",
+    "update": "FORCE_UPDATE=true run-p update:*",
```

Comment:
afaik, this syntax for setting an env var will not work on Windows. Is this a concern? I know we're being inconsistent about Windows dev environment support in different places, but I _think_ everything at least in this repo works on Windows now, and at some point in the past we went to the trouble of converting some shell scripts to use `shelljs`, specifically for Windows support.

Reply:
Ah yeah that's a good point. Sorry I never use non POSIX systems so it's easy to forget.

I think if we're not averse to it, we can use something like [cross-env](https://www.npmjs.com/package/cross-env) to fix this, unless you know of a better way?

Reply:
I'd like to avoid adding yet more dependencies (even dev ones) just for this. I was actually surprised the site still had `npm-run-all` installed - I don't think we're currently using that for anything. Is there anything especially bad about the setup we have now? Maybe we could just add a synchronous check to the site's `rollup.config.js` that looks for the files created by `npm run update` and bails early with a more helpful message.

Reply:
The reason I did this is because even as somebody who has done this a bunch of times before, the errors you get by not running this are just confusing. You then end up in the readme which you have said yourself is intimidating, so I figured that the best way around this is to automate.

`npm-run-all` is possibly unused, but is quite useful in this instance as these three tasks take a short while on my machine, and making them run in parallel is a vast improvement.

That being said, I'd be happy to pull out run-all, but I wouldn't want to drop this PR entirely - we should be making contributing easier and right now there's an awful lot of gymnastics you need to do in order to just run the site repo.

I'll experiment with other ways to set `FORCE_UPDATE` that don't require external deps, and see if there is a way to remove `npm-run-all`

Reply:
Does 'an awful lot' mean `npm ci`, `npm run update`, `npm run dev`? That's precisely one more line than people would probably tend to expect. I think most of the problems could be eliminated by having `rollup.config.js` print out a helpful error message and `process.exit` if it sees that the files created by `npm run update` are missing.

Reply:
I guess I'm referring to the cumulative set of things you might have to do following https://github.com/sveltejs/svelte/blob/master/site/README.md if you want to do some of the other things listed.

Doing a check for existence of the files produced by the set of scripts in `site/scripts` in the rollup config seems like an awkward, and brittle link, when really checking that the file is in a good state should really be the responsibility of the script which creates said file (and they get to share a variable holding the file's location, to boot).

I've updated the PR to remove `npm-run-all` and retain cross-platform support. see what you think.